andela/suyabay
# #116598305 Fix Suyabay User's count in admin dashboard

#### What does this PR do?
#### Description of Task to be completed?

Add pagination in user section for administrator
The total number of users displayed in the dashboard section is actually correct but the pagination navigation was ommited
#### How should this be manually tested?
1. Log in as an admin
2. navigate to the dashboard section.
3. See the number of users,
4. Click on `account` then `users`.  See the total number of users matches.
#### What are the relevant pivotal tracker stories?

116598305
#### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2016-03-30 at 13 57 05" src="https://cloud.githubusercontent.com/assets/16223627/14140025/677b4278-f67f-11e5-96e4-f44afce6cd96.png">

[@unicodeveloper](https://github.com/unicodeveloper)
